main.cpp

```diff
@@ -0,0 +1,99 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+using std::getline;
+
+const string CONFIG_FILE_WRITE_PATH = "/Users/Jon/Desktop/C++repos/CPT-180-27-Assignment-2-Config-Generator/config.txt";
+
+int main(int argc, char *argv[]){
+// init
```

Comment:
Right so you need a condition here in relationship to `argv` that will tell us if the command line argument was `init` or `edit`